services/accounting/controllers/send.ts

```diff
@@ -0,0 +1,57 @@
+const log = require("loglevel");
+const { checkUserHasSufficientSats } = require("@library/userHelper");
+const { validate } = require("uuid");
+const { processSplits } = require("@library/amp");
+import db from "@library/db";
+import asyncHandler from "express-async-handler";
+
+const createSend = asyncHandler(async (req, res: any, next) => {
+  const userId = req["uid"];
+  const request = {
+    contentId: req.body.contentId,
+    msatAmount: req.body.msatAmount,
+    comment: req.body.comment,
+    contentTime: req.body.contentTime ? req.body.contentTime : null,
+  };
+
+  if (request.msatAmount < 1000 || isNaN(request.msatAmount)) {
```

Comment:
nit - should flip these so we only compare a number to `< 1000`
```suggestion
  if (isNaN(request.msatAmount || request.msatAmount < 1000)) {
```